prod-cockpit-rcar.yaml

```diff
@@ -173,7 +169,7 @@ components:
         - [XEN_REL_pn-xen-tools, "4.16"]
         - [SRCREV_pn-xen, "d9703fe08c3d32dbfb89ab10502ad2ffeb52de62"]
 
-     layers:
+      layers:
```

Comment:
This is very suspicious change

Reply:
it is ok, previous position was wrong

Reply:
The change is still present in the last commit